frontend/src/components/User/User.js

```diff
@@ -1,78 +1,21 @@
+import '../../config';
 import React from "react";
 import axios from "axios";
-import Post from "../Post/Post";
 import { useState, useEffect} from "react";
```

Comment:
Mind having these imports on the same line as the `import React from "react";` line?